admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,542 @@
+link:../admin_guide/quota.html#configuring-quota-sync-period[configure the quota synchronization period], which controls how long to wait before restoring quota usage after resources are deleted.
+. If you want to remove an active quota to no longer enforce the limits of a project:
++
+----
+$ oc delete quota <quota_name>
+----
+
+*Now let's walk through that process and examine all the available options:*
+
+. To create project limits or quotas, you must first write a .yaml file that describes the resource that you want to restrict:
+
+* This *_LimitRange.yaml_* example explains all the
+link:../admin_guide/limits.html#container-limits[container limits] and
+link:../admin_guide/limits.html#pod-limits[pod limits] that you can place within
+your project:
++
+include::admin_guide/limits.adoc[tag=admin_limits_sample_definition]
++
+* These *_ResourceQuota.yaml_* examples explain all the
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Object Counts]
+and
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Compute Resources] that you can place within your project:
++
+include::admin_guide/quota.adoc[tag=admin_quota_object_counts]
++
+include::admin_guide/quota.adoc[tag=admin_quota_compute_resources]
++
+. Once you have defined your limit range object file, you can create a limit range:
++
+----
+$ oc create -f <limit_range_file> -n <project_name>
+----
++
+The `-n` flag is optional. Use it to create the limit in a different project
+than the one you currently are in.
+
+[[limit-number-projects]]
+=== Limiting the Number of Projects a User Can Have
+
+You can
+link:../admin_guide/managing_projects.html#limit-projects-per-user[limit the number of projects] that a user may request by categorizing users with label selectors with the `oc label` command. A label selector consists of the label name and the label value:
+
+----
+label=value
+----
+
+Once users are labeled, you must
+link:../admin_guide/managing_projects.html#modifying-the-template-for-new-projects[modify the default project template] in the *_master-config.yaml_* file
+link:../architecture/additional_concepts/admission_controllers.html[using an admission control plug-in]. This allows some users to create more projects than others, and you can define different values (or levels) for each label.
+
+*Use Case: You want to define 3 different levels for your users which define how many projects they can request.* The label is named `level`, and the possible values are  `bronze`, `silver`, `gold`, and `platinum`. Platinum users do not have a maximum number of project requests, gold users can request up to 10 projects, silver users up to 7 projects, bronze users up to 5 projects, and any users without a label are by default only allowed 2 projects.
+
+. Apply label selectors to users. For example, to apply the `level` label selector with a value of `bronze`:
++
+----
+$ oc label user <user_name> level=bronze
+----
++
+Repeat this step for the other levels.
```

Comment:
@bfallonf I did mean levels. Like once you've applied the bronze label to all your bronze users, then you need to apply the silver label, and after that the gold label. But you bring up a good point. I probably should indicate that this bronze labeling would need to be repeated for each bronze user.